script/i18n_lint.rb

```diff
@@ -100,6 +109,10 @@ def validate_content(yaml)
            !EXEMPTED_DOUBLE_CURLY_BRACKET_KEYS.include?(key)
         @errors[:invalid_interpolation_key_format] << key
       end
+
+      BANNED_PHRASES.keys.each do |banned|
+        @errors["banned_phrase_#{banned}"] << key if value.include?(banned)
```

Comment:
Oops, I missed the `downcase` from your implementation :blush: 
It should have been `value.downcase.include?(banned)`

Reply:
Ah yes, I'll do a followup 👀